### ISSUE TITLE
pageUp/Down gesture supported as pageUp/Down keys (or mouse wheel)

### ISSUE THREAD

**Issue by torappinfo** [termux/termux-app#101]
Since console apps has no knowledge of gestures, if we can convert pageUp/Down gestures to keys or mouse wheel events, it will enable more creative usages in termux.

For example, current gesture support is non-functional when using less in neovim terminal mode.

to reproduce the bug, enter neovim terminal mode with:
$ nvim -c term
Inside the neovim terminal mode, run 
$ less <some txt file>
Then pageUp/Down gestures trigger neovim quit terminal insert mode and thus make it nonfunctional.

Running neovim in linux, pressing pageUp/Down keys or mouse wheel works perfectly.

**Comment by fornwall** [comment]
What do you mean with 

> pageUp/Down gestures

here? Just scrolling up and down by dragging your finger on the screen?

**Comment by dvaldivia** [comment]
yes, in the context of claude code, it's better to scroll wit page up and down instead of arrows, so being able to change this setting would be supper useful
